<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberView</class>
 <widget class="QWidget" name="MemberView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="bannerBackground" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#bannerBackground {
	background-image: url(:/branding/images/brand/background.jpg);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <item>
       <widget class="QLabel" name="bannerLogo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bannerTitle">
        <property name="styleSheet">
         <string notr="true">#bannerTitle {
	color: rgb(255, 255, 255);
	font: 700 60pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>City Library</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="pageScroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1067</width>
        <height>700</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="memberTab">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="catalogueTab">
          <attribute name="title">
           <string>View Catalogue</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="tabContents">
             <item>
              <layout class="QVBoxLayout" name="filterContents">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Filter Search</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="searchContents">
               <item>
                <layout class="QHBoxLayout" name="searchBarLayout">
                 <item>
                  <widget class="QLineEdit" name="searchLineEdit"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="searchButton">
                   <property name="text">
                    <string>PushButton</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QListWidget" name="catalogueList"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="catalogueFooter">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="catalogueFooterLabel">
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="accountTab">
          <attribute name="title">
           <string>My Account</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QWidget" name="profileContent" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="profileLabel">
                  <property name="text">
                   <string>Profile</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="updateButton">
                  <property name="text">
                   <string>Update</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="2" column="2">
                 <widget class="QLabel" name="addressLabel">
                  <property name="text">
                   <string>Address :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="phoneLabel">
                  <property name="text">
                   <string>Phone :</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="emaiLabel">
                  <property name="text">
                   <string>Email : </string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="accountLabel">
                  <property name="text">
                   <string>Account # :</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLabel" name="phoneOutputLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="nameOutputLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="accountOutputLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QLabel" name="addressOutputLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QLabel" name="emailOutputLabel">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="verifiedLabel">
                  <property name="text">
                   <string>Verified</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="nameLabel">
                  <property name="text">
                   <string>Name :</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLabel" name="verfiedOutput">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="finesContent">
             <item>
              <widget class="QLabel" name="finesLabel">
               <property name="text">
                <string>Fines</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="finesList"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="issuedContent">
             <item>
              <widget class="QLabel" name="issuedLabel">
               <property name="text">
                <string>Issued</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="issuedList"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="holdContent">
             <item>
              <widget class="QLabel" name="holdLabel">
               <property name="text">
                <string>On Hold</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="holdList"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="historyContent">
             <item>
              <widget class="QLabel" name="historyLabel">
               <property name="text">
                <string>Past Books</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="historyList"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="accoutFooterBackground" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>110</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>110</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
              <item row="0" column="2">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="footerLogo">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>472</width>
                  <height>5</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
